Source/Core/AestraAudioController.cpp

```
@@ -169,6 +169,16 @@ void AestraAudioController::shutdown() {
 
 void AestraAudioController::setContent(std::shared_ptr<AestraContent> content) {
     m_content = content;
```

Cppcheck (2.20.0)

[performance] Function 'getTooltip()' should return member 'tooltipText_' by const reference.

(returnByReference)